packages/xo-web/src/common/render-xo-item.js

```diff
@@ -94,12 +96,14 @@ export const Host = decorate([
 Host.propTypes = {
   id: PropTypes.string.isRequired,
   link: PropTypes.bool,
+  memoryFree: PropTypes.bool,
   newTab: PropTypes.bool,
   pool: PropTypes.bool,
 }
 
 Host.defaultProps = {
   link: false,
+  memoryFree: true,
```

Comment:
IMO, it should be `false` by default.
Do you have a reason to make it `true` by default?

Reply:
to have it on `SelectHost` directly and I agree it isn't a good idea to use  `({ ...host, memoryFree: true })`.